src/yagooglesearch/__init__.py

```diff
@@ -144,7 +144,11 @@ def __init__(
         self.query = urllib.parse.quote_plus(query)
         self.tld = tld
         self.lang_html_ui = lang_html_ui
-        self.lang_result = lang_result.lower()
+        self.lang_result = (
```

Comment:
1. Since the text case is not standardized in `result_languages.txt`, and thus not standardized in `result_languages_list`, was thinking of just keeping this line as:

`self.lang_result= lang_result`

Thoughts?

2. For the PR, can you bump the version as well in line 17:

`__version__ = "1.10.0"`

Reply:
> Since the text case is not standardized in result_languages.txt, and thus not standardized in result_languages_list, was thinking of just keeping this line as: `self.lang_result= lang_result`

Personally, I also prefer this nice and clean solution. But this will result in a breaking change, since someone may not handle such lowercasing themselves before. If this breaking change is acceptable, let me know and I'll change to it soon.

> For the PR, can you bump the version as well in line 17: `__version__ = "1.10.0"`

Sure. I will add this version bump commit once the above changes are completed.